### DISCUSSION TITLE
Column names in table of M2M relation

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#5190] [Archive]
Hi,

I cannot seem to figure out how to change the column labels of a O2M table? I know how to add fields of the relation, but the column name I have no clue.

Any advice?

Regards,

Kees.

**Comment by rijkvanzanten** [comment]
The column labels are the field labels of the columns you have selected 🤔 Also, this might not be relevant anymore very soon now that https://github.com/directus/directus/pull/4093 is merged
